usaspending_api/disaster/v2/views/federal_account/spending.py

```diff
@@ -152,7 +152,8 @@ def total_queryset(self):
                     Case(
                         When(
                             self.final_period_submission_query_filters,
-                            then=F("obligations_incurred_by_program_object_class_cpe"),
+                            then=F("obligations_incurred_by_program_object_class_cpe")
+                            - F("deobligations_recoveries_refund_pri_program_object_class_cpe"),
```

Comment:
Is this suppose to be a minus for implementation? The ticket refers to adding the values together for DAIMS and subtracting for implementation. It looks like majority of the values are negative for this field so it would increment in most cases instead of decrement like mentioned in notes on the equation.

Reply:
Since this code is copy-paste it is easier to merge then fix all three endpoints together